Третье задание дипломного проекта и allure отчет

ZaurSeyfulaev opened the pull request.

ZaurSeyfulaev committed:
Третье задание дипломного проекта и allure отчет

pomegranate282922 commented on src/test/java/site/nomoreparties/stellarburgers/LoginTest.java:
```diff
@@ -0,0 +1,87 @@
+    @DisplayName("Вход в систему")
+    @Description("Вход в систему с использованием кнопки Войти в акаунт")
+    public void loginUsingSignInButtonTest() {
+        open("https://stellarburgers.nomoreparties.site");
```
Урлы выносим как константы и переиспользуем

pomegranate282922 commented on src/test/java/site/nomoreparties/stellarburgers/MainPageTest.java:
```diff
@@ -0,0 +1,57 @@
+        mainPage.clickFillingsMenuSection();
+        assertEquals(false, mainPage.checkSelectedButSection());
+        mainPage.clickBunMenuSection();
+        assertEquals(true, mainPage.checkSelectedButSection());
```
assertTrue и assertFalse

pomegranate282922 commented on src/test/java/site/nomoreparties/stellarburgers/RegisterNewUserTest.java:
```diff
@@ -0,0 +1,59 @@
+                userDataGenerator.getPassword(5));
+        assertEquals("Некорректный пароль", registerPage.getInvalidPasswordMessage());
+
+    }
```
А удалить созданного юзера?
Предусмотри и в негативном кейсе тоже.

pomegranate282922 reviewed (commented):
Все отлично - только добавить удаление юзера в кейсах на его создание.

ZaurSeyfulaev committed:
Исправления по замечаниям
- урлы вынес в константы
- добавил удаление новых пользователей
- заменил ассерты

pomegranate282922 reviewed (approved):
Зачёт!
